src/bp/core/config/botpress.config.ts

```diff
@@ -63,4 +63,18 @@ export type BotpressConfig = {
    */
   licenseKey: string
   allowStats: boolean
+  /**
+   * When this feature is enabled, fields saved as user attributes will be automatically erased when they expires. The timer is reset each time the value is modified
+   * Setting a policy called "email": "30d" means that once an email is set, it will be removed in 30 days, unless it is changed in that timespan
+   */
+  dataRetention?: DataRetentionConfig
+}
+
+export interface DataRetentionConfig {
+  janitorInterval: string
+  policies: RetentionPolicy[]
+}
+
+export type RetentionPolicy = {
+  [key: string]: string
```

Comment:
Could you perhaps just add a small description / `@example` for it to be included in the documentation automatically? e.g. `@example for key "user.identity.email"`